README.md

```diff
@@ -1 +1,38 @@
 # Haskell Errors
+
+So you would like to help improve the error messages we see in the Haskell toolchain and ecosystem?
+
+Welcome! We're glad to have you, and we will do our best to ensure your efforts are worthwhile.
+
+
+## Project Goals
+
+* Serve the Haskell community, focusing on improvements to the error messages seen in the Haskell ecosystem.
+
+* Lower Haskell's barrier-to-entry by increasing the general understanding of the error messages experienced during development.
+
+
+## What and How?
+
+This project is taking shape, so many details are still being worked out, however:
```

Comment:
Is there a good way to get a notification in, say, 6 months to revisit? I'm leery of making time-sensitive statements like this in a repo, where it may continue to live on in 10 years. The GHC Steering Committee makes good use of https://github.com/probot/reminders.

Reply:
Sure, that sounds like a nice solution to the problem. I've installed/activated the reminders app for this repo.

Reply:
/remind me to review this comment for updates in 3 months

Reply:
My experience with the reminder-bot is that it speaks up when it snags a comment. Maybe it looks only at top-level comments, not ones in reviews? I'm pretty sure this one has gone unnoticed.